src/data/faq.json

```diff
@@ -71,8 +71,9 @@
-                                    "See FAQ: <ul><li><a href='#risk_encounter_not_realtime' target='_blank'>Why do I receive a risk warning several days after the risk encounter?</a></li><li><a href='#risk_encounter_no_warning' target='_blank'>Why don't I get a risk encounter warning even though I've had contact with someone who has a risk encounter displayed in their app?</a></li><li><a href='#risk_encounter_different_devices' target='_blank'>How do differences in risk assessment occur on multiple devices, even if the respective smartphones were not far apart?</a></li><li><a href='#risk_encounter_doubted' target='_blank'>Why do I receive a risk warning for a date when I had no contacts / did not leave the house?</a></li><li><a href='#risk_encounter_place' target='_blank'>Are people and/or places assigned to a risk encounter?</a></li></ul>"
+                                    "<b>Updated ?????r</b>",
+                                    "In order to receive warnings, there are certain requirements that must be met:<ul><li>After receiving the positive test result, it must be voluntarily shared by the user via the Corona-Warn-App. If the test result could not be determined via the app, the positive PCR test result can be entered and shared via a TAN. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App.</li><li>The app must have been installed and active on both devices <b>before</b> sharing the positive test result via the Corona-Warn-App, otherwise no encounters can have been recorded.</li><li>The two devices must have been close enough and long enough <b>before</b> sharing the test result. Encounters <b>after</b> sharing a positive result are of course <b>not</b> warned.</li><li>The time period for the onset of the first symptoms given when sharing the test result, as well as the type or intensity of the symptoms, may also play a role in the risk calculation. So it also depends on which days before sharing the test result the person's devices recorded the encounters. In some cases, this may result in no warning being sent.</li></ul>",

```

Comment:
In
"Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App."

"teleTAN" = "a teleTAN"